docs/content/configuration/csi_ug_config_create_replication.md

```diff
@@ -0,0 +1,45 @@
+# Creating a VolumeReplication
+
+Create a VolumeReplication YAML file to replicate a specific PersistentVolumeClaim (PVC).
+
+VolumeReplicationClass needs to be present before a VolumeReplication can be created. For more information, see [Creating a VolumeReplicationClass](csi_ug_config_create_vol_replicationclass.md).
+
+**Note:** Replication is referred to as the more generic volume replication within this documentation set. Spectrum Virtualize products refer to this feature as the remote copy function.
+
+When replicating a volume, be sure to follow all of the replication configurations, found in [Compatibility and requirements](../installation/csi_ug_requirements.md) before volume replication.
+
+1.  Replicate a specific PersistentVolumeClaim (PVC) using the `demo-volumereplication.yaml`.
+
+    For more information about PVC configuration, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
+
+    ```
+    apiVersion: replication.storage.openshift.io/v1alpha1
+    kind: VolumeReplication
+    metadata:
+      name: demo-volumereplication
+      namespace: default
+    spec:
+      volumeReplicationClass: demo-volumereplicationclass
+      replicationState: primary
+      dataSource:
+        kind: PersistentVolumeClaim
+        name: demo-pvc-file-system  # Ensure that this is in the same namespace as VolumeReplication.
+    ```
+
+2.  After the YAML file is created, apply it by using the `kubectl apply -f` command.
+
+    ```
+    kubectl apply -f <filename>.yaml
```

Comment:
since there is only 1 yaml in this page, IMO, it would be better to specify its explicit name (as was done in most of the other single-yaml pages)
```suggestion
    kubectl apply -f demo-volumereplication.yaml
```
(same for [other](https://github.com/IBM/ibm-block-csi-driver/blob/8d18ae5a6d95b15894c814e59a27c133aea06d8a/docs/content/configuration/csi_ug_config_create_snapshots.md) places)

Reply:
as I wrote [here](https://github.com/IBM/ibm-block-csi-driver/pull/360#discussion_r713795764)
I think that it's better that we keep the commands more generic. It's better not to specify in this case because the others are examples and the customer may be using other names for for their files. this way we know that they will apply the correct name for the command.

Reply:
most of the single-yaml pages are not like this, and IMO it is more important for them to be aligned than to have the "better" convention

Reply:
i've updated a lot (I think all at this point) instances for consistency. the better convention here is to give the general command/ information and not a specific example. the one exception is in the sample usage section.
DONE

Reply:
speaking of aligning this - why is there no `the X created message is emitted` line in the snapshot+replication (+classes+topology) pages?

Reply:
I need the messages given to me. please send me the exact wording and I'll add it in.

Reply:
@tyichye19 should have them

Reply:
For VolumeReplicationClass - `volumereplicationclass.replication.storage.openshift.io/<volumereplicationclass-name> created`
For VolumeReplication - `volumereplication.replication.storage.openshift.io/<volumereplication-name> created`
For VolumeSnapshotClass - `volumesnapshotclass.snapshot.storage.k8s.io/<volumesnapshotclass-name> created`
For VolumeSnapshot - `volumesnapshot.snapshot.storage.k8s.io/<volumesnapshot-name> created`

Reply:
DONE

Reply:
the [volumesnapshot](https://github.com/IBM/ibm-block-csi-driver/blob/c4cde796d28b2347bfe3484a509edbb9bce534bc/docs/content/configuration/csi_ug_config_create_snapshots.md) and [volumereplication](https://github.com/IBM/ibm-block-csi-driver/blob/c4cde796d28b2347bfe3484a509edbb9bce534bc/docs/content/configuration/csi_ug_config_create_replication.md) message lines should be at the end of the 2nd step (not 3rd).
in addition, I [don't see it](https://github.com/IBM/ibm-block-csi-driver/blob/c4cde796d28b2347bfe3484a509edbb9bce534bc/docs/content/configuration/csi_ug_config_create_vol_snapshotclass_topology.md) in the [snapshot class topology](https://github.com/IBM/ibm-block-csi-driver/pull/360#discussion_r713920958) page

Reply:
@roysahar-ibm  @ArbelNathan  can you give me the created message line for snapshot class topology?
@tyichye19  doesn't have it. thanks

Reply:
@oriyarde  thanks. I fixed the placement of the two files.

Reply:
it should be the same as VolumeSnapshotClass

Reply:
placement LGTM

Reply:
DONE